sdk/machinelearningservices/Azure.ResourceManager.MachineLearning/src/Generated/Models/MachineLearningComponentVersionProperties.Serialization.cs

```diff
@@ -187,7 +180,6 @@ internal static MachineLearningComponentVersionProperties DeserializeMachineLear
                 {
                     if (property.Value.ValueKind == JsonValueKind.Null)
                     {
-                        componentSpec = null;
```

Comment:
this change is also fine, because this local variable is initialized as `default` which is `null` in the context because it is a reference type.